rerun_py/Cargo.toml

```
@@ -80,7 +80,7 @@ numpy.workspace = true
 once_cell.workspace = true
 parking_lot.workspace = true
 pyo3 = { workspace = true, features = [
-  "abi3-py38",
+  "abi3-py39",
```

Let's make it easier to find with a text search once we want to update the minimum supported Python version again:
```suggestion
  "abi3-py39",  # Require at least Python 3.9
```